frontend/components/InteractiveInputs/InteractiveInputs.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import ImageSlider from "../InteractiveImage/ImageSlider";
+import HolonButton from "../VersionOne/Buttons/HolonButton";
+import s from "./InteractiveInputs.module.css";
+
+type Props = {
+  input: {
+    id?: number;
+    name?: string;
+    type?: string;
+    animationTag?: string;
+    options: InteractiveInputOptions[];
+  };
+};
+export type InteractiveInputOptions =
+  | {
+      id?: number;
+      option: string;
+    }
+  | {
+      slider_value_default?: number;
+      slider_value_min?: number;
+      slider_value_max?: number;
```

Comment:
I noticed the casing of these props isn't consistent with those in SectionBlock. Is that intentional? I suspect the types in SectionBlock might not be needed.

SecionBlock.tsx:

```tsx
export type InteractiveInputOptions =
  | {
      id?: number;
      option: string;
    }
  | {
      sliderValueDefault?: number;
      sliderValueMax?: number;
      sliderValueMin?: number;
    };
```